merlin-server/sql/merlin/tables/activity_type.sql

```diff
@@ -4,6 +4,7 @@ create table activity_type (
   parameters merlin_parameter_set not null,
   required_parameters merlin_required_parameter_set not null,
   computed_attributes_value_schema jsonb,
+  subsystem text null,
```

Comment:
🔻 I'd like to double check with @bashbash - I know we had discussed using the term `category` to be more generic than `subsystem`, and I'm unclear on what decision was made about that.

Reply:
Activity Types specifically have a subsystem (or potentially several, but @bashbash suggested we start with one) rather than a category.

Reply:
Could you provide a reference to where that decision was documented? I'd like to get product owner sign-off on this prior to merge

Reply:
@joswig or @bashbash Can you comment here please?

Reply:
I'd like to go with category, I'm good with limiting it to zero or one category for the initial implementation

Reply:
I can make the change, but in answer to your original question, @mattdailis, ActivityTypes using `subsystem` was documented in [this Figma](https://www.figma.com/file/JsDIKVE0fR3mfiLYDJ01FM/AERIE-Metadata-Discussion-%23575?type=whiteboard&node-id=27-6010&t=7mvyRuLg7i5qh05f-0), in the part circled in blue on the right.

Reply:
It is indeed a category but I would call it sub-system, category is way too generic is my 2 cents